### ISSUE TITLE
Linked Data Entity - Dynamic collection view or portal view or landing page

### ISSUE THREAD

**Issue by Natkeeran** [Islandora-CLAW/CLAW#856]
| Title (Goal)  | Create a dynamic portal or landing page for linked data entities
--------------- |------------------------------------                           |
| Primary Actor | End user|
| Scope         | Drupal View/Module |
| Level         | Low |
| Story         | As an end user, I would like to see all content related to a linked data entity in a portal or collection view.  For example, if I go to a place entity, I would like to see all people from that place, all books that have this place as subject, all organizations located in this place etc.

**Comment by Natkeeran** [comment]
@bryjbrown 

Yes, I should have been more careful with the word entity.  I meant all the "things" in an ontology that will have a uri (rdf resources, types!).  For example, in the schema.org, they would include CreativeWork, Person, Organization, Place, Event etc. 

What I am suggesting is creating a view that when the user goes to an instance of type, they get to browse all the other things (paged) it is linked to.  

Examples:
https://tools.wmflabs.org/reasonator/?&q=172 (Place)
https://tools.wmflabs.org/reasonator/?q=Q76 (Person)

This type of portal/view will include authorities but not restricted to them.  Anything with a uri can potentially have such a view.  

Previously, if we need to collect heterogeneous data about something in 7.x, we would have to create a collection.  I think with the linked data approach, it can be dynamically generated.

**Comment by seth-shaw-unlv** [comment]
My [migrate proof-of-concept](https://github.com/seth-shaw-unlv/claw-migrate-files-poc) implements and example of this. After import each image has an entity link to applicable subjects (each subject term is a node) and the subject nodes have a [view-block](https://github.com/seth-shaw-unlv/claw-migrate-files-poc/blob/master/unlv_image/config/install/views.view.subject_images.yml) listing all the images that link to that subject node. 

It is pretty simple to build using the view interface.

**Comment by Natkeeran** [comment]
@seth-shaw-unlv 

I was able to import the nodes.  But, I cannot quite test this out because of the node has term issue related to placing the block.  As you noted, a lot of the functionality can be build using views.

**Comment by seth-shaw-unlv** [comment]
Okay. I hadn't looked at it since the giant content model overhaul was finally committed, so there may have been a change along the way that broke it. I will have to play with it later.

**Comment by seth-shaw-unlv** [comment]
Oh, @Natkeeran, if you wouldn't mind posting what you saw to the [proof-of-concept issues](https://github.com/seth-shaw-unlv/claw-migrate-files-poc/issues), I would appreciate it.

**Comment by Natkeeran** [comment]
@seth-shaw-unlv 

I don't think the issue is related to the proof-of-concept module.  It is the original issue you found related to that big PR where you cannot place any block without specifying a term.  We decided to address the issue after PR gets in, but not sure if there is a ticket to followup on that issue.
